pages/understanding-json-schema/structuring.md

```diff
@@ -1,8 +1,7 @@
 ---
 title: "Structuring a complex schema"
 section: docs
----
-
+--- 
```

Comment:
Why editing this file?

Reply:
I think its done by my prettier extension , I will resolve this 😅